### ISSUE TITLE
Overlapping graphs

### ISSUE THREAD

**Issue by panhans** [Kibibit/hass-kibibit-theme#12]
Hi there,

I have an issue with graphs in Firefox and Chrome. The corners are coloured. That means the graph is overlapping the actual card. Is this a bug or is there a workaround for this?

Thx in advance!

**Comment by MrSleeps** [comment]
The svg graphs need the the bottom border added to the css

**Comment by panhans** [comment]
Yes, manupulating CSS with firefox dev-tools worked fine. Seems like hui-graph-header-footer-border-radius hast no impact.
Is this an issue with the theme or with lovelace itself?

**Comment by Thatkookooguy** [comment]
I'm not sure I understand exactly what you're facing.... can you try and give more details? what are you trying to do and what is the problem? does this happen just by adding a mini graph card?

**Comment by masminl** [comment]
![Screenshot_20220911_113647_Home Assistant](https://user-images.githubusercontent.com/28845289/189520948-0b80b280-c4b6-43ac-bbed-4825e8da46af.jpg)
I really like your theme and want to use it for the wallpanel I'm building. 
Is there any chance you can fix this glitch? It happens on both my Android tablet and IPhone.
Happy to provide further info and test!
